vhdbuilder/packer/init-variables.sh

```diff
@@ -218,10 +218,6 @@ if [[ "$OS_SKU" == "CBLMariner" ]]; then
 				--os-type Linux
 
 			echo "Creating new image-definition for imported image ${IMPORTED_IMAGE_NAME}"
-			if [[ ${ENABLE_TRUSTED_LAUNCH} == "True" ]]; then
```

Comment:
this was redundant. packer VMs for kata are not based of marketplace image, but from an image built from imported VHD. security type for packer VM cannot be set to TL, the image definition doesnt need to have TL as well, since the final VHD gets stored to the appropriate definition.